homeassistant/bootstrap.py

```diff
@@ -371,8 +366,3 @@ def _ensure_loader_prepared(hass: core.HomeAssistant) -> None:
     """Ensure Home Assistant loader is prepared."""
     if not loader.PREPARED:
         loader.prepare(hass)
-
-
-def _mount_local_lib_path(config_dir: str) -> None:
```

Comment:
Let's just rename this method to not have the `_` prefix and import it from here into the script. However, the 'weird' part now is that the script will now need to be passed in a config dir or it will create the default config dir. Anyway, I think that is fine for now .

An alternative could be to use a temporary directory to install script dependencies. However that might just get very messy.